### PR TITLE
Changed parameters order of model.document.createRoot() method.

### LINKED ISSUE #390: Default root name
We need a default name of the root, at least for the tests consistent. @Reinmar suggested `main` what is fine for me. I think we could go further and set it as a default parameter of the `(model|view).document.(get|create)Root`. Then a single root editor creator would not need to care about the root name.

### PR DESCRIPTION
**Pull request by oskarwrobel**
Fixes #390 

Related:
https://github.com/ckeditor/ckeditor5-typing/pull/11
https://github.com/ckeditor/ckeditor5/pull/200

### CODE REVIEW

File: tests/conversion/advanced-converters.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ let modelDoc, modelRoot, viewRoot, mapper, writer, modelDispatcher, viewDispatch
 
 beforeEach( () => {
 	modelDoc = new ModelDocument();
-	modelRoot = modelDoc.createRoot( 'root' );
+	modelRoot = modelDoc.createRoot( '$root', 'root' );
```
Comment (pjasiun) [inline tests/conversion/advanced-converters.js]: The "root" name was introduced before we had the default, `main` root name. Now it should be removed.

File: tests/datacontroller.js [hunk 2]
```diff
@@ -23,8 +23,8 @@ describe( 'DataController', () => {
 
 	beforeEach( () => {
 		modelDocument = new ModelDocument();
-		modelDocument.createRoot( 'main' );
-		modelDocument.createRoot( 'title' );
+		modelDocument.createRoot( '$root', 'main' );
```
Comment (szymonkups) [inline tests/datacontroller.js]: Can be changed to `modelDocument.createRoot()`.

File: tests/conversion/advanced-converters.js (hunk 1 above)
Comment (szymonkups) [inline tests/conversion/advanced-converters.js]: In tests that use `doc.createRoot( '$root', 'root' );` we should use simpler version `doc.createRoot()` and eventually fix places where `root` name is used to use `main`. Is that correct @pjasiun?

File: src/model/document.js [hunk 3]
```diff
@@ -181,13 +181,13 @@ export default class Document {
 	/**
 	 * Creates a new top-level root.
 	 *
-	 * @param {String} [rootName='main'] Unique root name.
 	 * @param {String} [elementName='$root'] Element name. Defaults to `'$root'` which also have
+	 * @param {String} [rootName='main'] Unique root name.
 	 * some basic schema defined (`$block`s are allowed inside the `$root`). Make sure to define a proper
```
Comment (szymonkups) [inline src/model/document.js]: This text should also be moved.

File: tests/conversion/advanced-converters.js (hunk 1 above)
Comment (pjasiun) [inline tests/conversion/advanced-converters.js]: Yep. Exactly.

**Pull request merged**